.github/actions/setup_ubuntu_build/action.yml

```diff
@@ -24,10 +24,20 @@ runs:
   steps:
     - name: Install prerequisites
       shell: bash    
+      env:
+        UBUNTU_VER : '${{ inputs.ubuntu_version }}'
+        ARCH : '${{ inputs.arch }}'
       run: |
-        sudo apt-get install -y spirv-tools
-        pip install lit
-
+        pip install lit clang-format==17.0.6 virtualenv
+        if [ "$ARCH" = "x86" ]; then sudo dpkg --add-architecture i386 ; fi
+        wget -qO - https://packages.lunarg.com/lunarg-signing-key-pub.asc | sudo apt-key add -
+        if [ "$UBUNTU_VER" = "20.04" ]; then sudo wget -qO /etc/apt/sources.list.d/lunarg-vulkan-1.3.243-focal.list https://packages.lunarg.com/vulkan/1.3.243/lunarg-vulkan-1.3.243-focal.list; fi
+        if [ "$UBUNTU_VER" = "22.04" ]; then sudo wget -qO /etc/apt/sources.list.d/lunarg-vulkan-1.3.243-jammy.list https://packages.lunarg.com/vulkan/1.3.243/lunarg-vulkan-1.3.243-jammy.list; fi
+        if [ "$UBUNTU_VER" = "24.04" ]; then sudo wget -qO /etc/apt/sources.list.d/lunarg-vulkan-noble.list https://packages.lunarg.com/vulkan/lunarg-vulkan-noble.list; fi
+        sudo apt-get update
+        sudo apt-get install --yes spirv-tools doxygen
```

Comment:
I am pretty sure we don't need spirv-tools if we install vulkan.

Reply:
You're right. Removed.